app.test.js

```diff
@@ -19,4 +19,84 @@ describe('Server', () => {
     });
   });
 
+  describe('GET /api/v1/projects/:id', () => {
+    it('should return a happy status code of 200 and a single project object if the project exists', async () => {
+      const expectedProject = await database('projects').first();
+        delete expectedProject.created_at;
+        delete expectedProject.updated_at;
+      const { id } = expectedProject;
+
+      const response = await request(app).get(`/api/v1/projects/${id}`);
+      const result = response.body[0];
+        delete result.created_at;
+        delete result.updated_at;
+
```

Comment:
I used the delete code here because the values for the created_at / updated_at properties were mismatched. The values returned from Supertest versus our Database were mismatched with regard to type (string vs integer), which was causing tests to fail.

An alternative to deleting the properties would be the card below (credit to Allie). Stringifying then parsing the SuperTest response fixes the mismatch with the DB format.

```
const cleanedProject = JSON.parse(JSON.stringify(expectedProject))
```

Reply:
I actually had two questions typed out about the delete lines before I saw this explanation lol. Thank you for clarifying what is taking place here